doc/nrf/known_issues.rst

```diff
@@ -1346,101 +1372,179 @@ nRF5340 Audio
 
 The issues in this section are related to the :ref:`nrf53_audio_app` application.
 
-.. rst-class:: v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0
 
-OCT-2501: Charging over seven hours results in error
-  Since the nRF5340 Audio DK uses a large battery, the nPM1100 can go into error when charging time surpasses 7 hours.
-  This is because of a protection timeout on the PMIC.
-  The charging is stopped when this error occurs.
+OCT-2070: Detection issues with USB-C to USB-C connection
+  Using USB-C to USB-C when connecting an nRF5340 Audio DK to PC is not correctly detected on some Windows systems.
 
   **Affected platforms:** nRF5340 Audio DK
 
-  **Workaround:** To start the charging again, turn the nRF5340 Audio DK off and then on again.
+.. rst-class:: v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
 
-.. rst-class:: v2-3-0
+OCT-2154: USB audio interface does not work correctly on macOS
+  The audio stream is intermittent on the headset side after connecting the gateway to a Mac computer and starting audio stream.
+  This issue occurs sporadically after building the nRF5340 Audio application with the default USB port as the audio source.
 
-OCT-2472: Headset fault upon gateway reset in the bidirectional stream mode
```

Comment:
We should not remove OCT-2472 but only shows the issue happens on v2.3.0 and doesn't go into v2.4.0